app/assets/javascripts/discourse/app/widgets/post.js

```diff
@@ -280,21 +280,6 @@ createWidget("post-meta-data", {
       );
     }
 
-    const lastWikiEdit =
-      attrs.wiki && attrs.lastWikiEdit && new Date(attrs.lastWikiEdit);
-    const createdAt = new Date(attrs.created_at);
-    const date = lastWikiEdit ? dateNode(lastWikiEdit) : dateNode(createdAt);
-    const attributes = {
-      class: "post-date",
-      href: attrs.shareUrl,
-      "data-share-url": attrs.shareUrl,
-      "data-post-number": attrs.post_number,
-    };
-
-    if (lastWikiEdit) {
-      attributes["class"] += " last-wiki-edit";
```

Comment:
We probably shouldn't remove the `last-wiki-edit` class here, it doesn't feel like it goes with the share-popup change?

Reply:
This was moved to the new `post-date` widget below.